cypress/integration/features/files/myFiles.feature

```diff
@@ -0,0 +1,56 @@
+@files @stable_test
+Feature: Files Overview - To use LibreOffice for text, calc and presentation document
+
+        As a user I want to create and edit a LibreOffice text document so that I write into it
+
+Scenario: Create a LibreOffice text document
+        Given I am logged in as a 'student' at 'brb'
+        When I go to files overview
+        And I go to personal files overview
```

Comment:
Is it ok to use ADD? I thought that it will not be supported in future by Cypress ( I think you told me that).